src/Service.Tests/Unittests/ConfigFileWatcherUnitTests.cs

```diff
@@ -65,8 +65,11 @@ HostMode mode
         /// Use the file system (not mocked) to create a hot reload
         /// scenario of the REST runtime options and verify that we
         /// correctly hot reload those options.
+        /// NOTE: This test is ignored until we have the possibility of turning
```

Comment:
On one of your hot reload issues (or a new one that tracks enabling hot reload with a feature switch), point to this file::line::testmethod to ensure that this test is re-enabled when appropriate.

Reply:
issue here https://github.com/Azure/data-api-builder/issues/2109